### ISSUE TITLE
Custom loaded model is not working properly 

### ISSUE THREAD

**Issue by prudhviraju535** [ThilinaRajapakse/simpletransformers#441]
**Describe the bug**
HI, I am trying to load the custom NER model which is trained on GPU to CPU. but after loading the model and doing prediction on new data. every entity is going into one category. But the model which i trained on CPU and loaded in cpu is working fine , when i saved model in h5 format also.

**To Reproduce**

train_args = {"classification_report": False, "adam_epsilon": 1e-08,"model_name": "emilyalsentzer/Bio_ClinicalBERT", "model_type": "bert", "do_lower_case": False, "early_stopping_consider_epochs": False, "learning_rate": 4e-05, "logging_steps": 50}

model = NERModel('bert','Model_out/',use_cuda=False,args=train_args,
                 labels= ['I-ROA', 'B-IND', 'I-AT', 'I-SOA', 'B-ROA', 'I-IND', 'I-VB', 'B-SOA', 'B-VB', 'O', 'B-AT'])

**Expected behavior**

Is there a way to load a GPU custom trained model  in CPU  or vise versa. The model which i loaded is prediction every word into one category. i want to save the model for deployment and testing.
Error : 
[{'PLACE': 'B-IND'}, {'5': 'B-IND'}, {'GM': 'B-IND'}, {'1': 'B-IND'}, {'FULL': 'B-IND'}, {'APPLICATORFUL': 'B-IND'}, {'INTRAVAGINALLY': 'B-SOA'}, {'EVERY': 'B-IND'}, {'DAY': 'B-IND'}, {'FOR': 'B-IND'}, {'5': 'B-IND'}, {'DAYS': 'B-IND'}]
**Screenshots**

**Desktop (please complete the following information):**
 - OS - Ubuntu (trained)
Windows 10 (model loading )

**Additional context**

**Comment by ThilinaRajapakse** [comment]
It shouldn't really matter if the model was trained on the GPU or the CPU. Could it possibly be an issue with changing the OS? What happens if you load the model on Ubuntu without using the GPU?

**Comment by prudhviraju535** [comment]
Hi, 
I have tested it on same env (Ubuntu) by loading the model from output dir, i am getting the same issue again. is there any other way to save the model and load like torch. save ?

model = NERModel('bert','outputs/',use_cuda=True,args=train_args, labels= ['I-ROA', 'B-IND', 'I-AT', 'I-SOA', 'B-ROA', 'I-IND', 'I-VB', 'B-SOA', 'B-VB', 'O', 'B-AT'])

Model output:
PRED,_ = model.predict(['PLACE 5 GM 1 FULL APPLICATORFUL INTRAVAGINALLY EVERY DAY FOR 5 DAYS'])

PRED=[[{'PLACE': 'I-IND'}, {'5': 'I-IND'}, {'GM': 'I-IND'}, {'1': 'I-IND'}, {'FULL': 'I-IND'}, {'APPLICATORFUL': 'I-IND'}, {'INTRAVAGINALLY': 'I-IND'}, {'EVERY': 'I-IND'}, {'DAY': 'I-IND'}, {'FOR': 'I-IND'}, {'5': 'I-IND'}, {'DAYS': 'I-IND'}]]
